src/borg/archiver.py

```diff
@@ -2616,9 +2620,17 @@ def process_epilog(epilog):
                                help="""specify format for file listing
                                 (default: "{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NL}")""")
         subparser.add_argument('--json', action='store_true',
-                               help='format output as JSON. The form of --format is ignored, but keys used in it '
-                                    'are added to the JSON output. Some keys are always present. Note: JSON can only '
-                                    'represent text. A "bpath" key is therefore not available.')
+                               help='Only valid for listing archives. Format output as JSON. '
```

Comment:
maybe change this to "listing repo contents" (-> archives) to make it more consistent with the phrasing below.
it is also more consistent internally as this is triggered by a repo-only argument.